aws/templates/solution/solution_rds.ftl

```diff
@@ -117,23 +132,22 @@
                         "}",
                         "pseudo_stack_file=\"$\{CF_DIR}/$(fileBase \"$\{BASH_SOURCE}\")-pseudo-stack.json\" ",
                         "create_deploy_snapshot || return $?" 
-                    ] +
-                    configuration.Encrypted?then(
-                        [
-                            "# Encrypt RDS snapshot",
-                            "function convert_plaintext_snapshot() {",
-                            "info \"Checking Snapshot Encryption... \"",
-                            "encrypt_snapshot" + 
-                            " \"" + region + "\" " +
-                            " \"" + rdsPreDeploySnapshotId + "\" " +
-                            " \"" + segmentKMSKey + "\" || return $?",
-                            "}",
-                            "convert_plaintext_snapshot || return $?"
-                        ],
-                        []
-                    )
-                /]
-            [/#if]
+                    ],
+                    []) +
+                (configuration.Backup.SnapshotOnDeploy && configuration.Encrypted)?then(
```

Comment:
Thought this was already merged?

Reply:
Moved it around, the whole prologue subset was wrapped in an if loop for the snapshot on deploy. Felt better that the deployment subset was always there and the necessary sections were added.